tests/Resources/RHOSi.resource

```diff
@@ -27,6 +27,7 @@ RHOSi Setup
     ...                Do Not extend this keyword for high-level setup, e.g., don't open browser
     Protect Sensitive Variables In Keywords
     Suite Availability Setup    ${RHODS_PROMETHEUS_URL}    ${RHODS_PROMETHEUS_TOKEN}
+    Initialize Global Variables
```

Comment:
I think this keyword should be after `Protect Sensitive Variables In Keywords`

Reply:
done